sw/device/silicon_creator/manuf/data/manuf_testplan.hjson

```diff
@@ -0,0 +1,408 @@
+            TODO: Should the transport image contain silicon creator pubilc keys to be able to
+            verify the certificate before install?
+
+            - Pre-load the OTP with an image of a personalized device.  See
+              manuf_ft_provision_rma_token_and_personalization for more details.
+            - The device flash should be loaded with the transprot image. See
+              manuf_ft_load_transport_image for details.
+            - The host sends a command to the device including the device certificate.
+            - The device verifies that the certificate corresponds to its device key.
+            - The certificate is then stored into a flash page (info versus regular flash TBD).
+            - Send response to the host with test result.
+            '''
+      tags: ["dv", "fpga", "silicon"]
+      stage: V3
+      tests: []
+    }
+
+    {
+      name: manuf_ft_eom
+      desc: '''End of manufacturing.
+
+            This test requires the device to have completed the manuf_ft_load_certificates test.
+            - The host sends a command to the device to trigger the end of manufacturing.
+            - Final OTP configuration options are set.
+            - Manufacturing functionality supported by the ROM_EXT and transport image will be
+              disabled after reset.
+            - Trigger software reset.
+            - The host sends a command to ensure device is in post-manufacturing state.
+            '''
+      tags: ["dv", "fpga", "silicon"]
+      stage: V3
+      tests: []
+    }
+
+    {
+      name: manuf_rma_entry
+      desc: '''Verifies RMA entry.
+
+            In silicon, the entire process is executed in a secure environment in adherence to
+            certification requirements.
+
+            - Initialize device to reflect eom state. See manuf_ft_eom for more details.
+            - Select LC TAP via strap pins.
+            - Read DEVICE_ID via LC TAP interface.
+            - Powerdown the device.
+
+            The Silicon Creator is expected to lookup the RMA_UNLOCK_TOKEN in a manufacturing database
+            using the DEVICE_ID and decrypt it at an offline location. The token cannot leave the
+            secure environment in unencrypted form at any time, implying that there is a way for the
+            secure environment to handle the decryption operation.
+
+            - Select LC TAP via strap pins.
+            - Power on the device.
+
+            The following steps are performed via the LC TAP interface:
+            - Read DEVICE_ID. Confirm expected response.
+            - Load RMA_UNLOCK_TOKEN and request transition to RMA state.
+            - Poll for status completion.
+            - Reset device by toggling POR pin.
+
+            RMA entry will trigger a rotation of scrambling keys. It is recommended to either
+            connect to the CPU JTAG to continue debug, or bootstrap the device with new a firmware
```

Comment:
Yeah, that could be tricky. I wonder if inserting a short busy-loop before shutdown would increase our probability of halting the device.

Reply:
Possibly, shutdown is after bootstrap but if bootstrap is disabled, this will require attaching and disabling the watchdog right away. We should probably have a testpoint for this too (with a max number of attempts).

Reply:
It looks like we are better at halting the core now thanks to @dmcardle.